frontend/src/layout/navigation-3000/sidepanel/SidePanel.tsx

```diff
@@ -150,15 +152,24 @@ export function SidePanel(): JSX.Element | null {
         ? [
               {
                   title: 'Open in side panel',
-                  items: extraTabs.map((tab) => {
-                      const { Icon, label } = SIDE_PANEL_TABS[tab]
-
-                      return {
-                          label: label,
-                          icon: <Icon />,
-                          onClick: () => openSidePanel(tab),
-                      }
-                  }),
+                  items: [
+                      ...extraTabs.map((tab) => {
+                          const { Icon, label } = SIDE_PANEL_TABS[tab]
+
+                          return {
+                              label: label,
+                              icon: <Icon />,
+                              onClick: () => openSidePanel(tab),
+                          }
+                      }),
+                      // FIXME: This is a off ramp for the feature previews moving from the side panel to the settings page, this shouldn't be here for long.
+                      {
+                          label: 'Feature previews',
+                          icon: <IconFeatures />,
+                          onClick: () => router.actions.push(urls.settings('user-feature-previews')),
+                          tooltip: 'Feature previews has moved to the settings page',
```

Comment:
I don't think people will see this tooltip. I think we should show them where it is by 
- When they click this button in the right side panel
- open the user menu 
- highlight the button there
- That button has a tooltip that shows automatically that says "feature previews are now accessed from here"
- They click it there (to reinforce the new pattern)
- It disappears from the right side panel (persisted value in kea logic)

The confusing part isn't that the feature previews are on the settings page. The confusing part is how you get there.